server/controllers/food.js

```diff
@@ -70,7 +69,7 @@ export default {
    * List of Food categories
    */
   async list(req, res) {
-    authorizeByRole(req.user.roles, [CUSTOMER, INVENTORY, SCHEDULE])
+    authorizeByRole(req.user.roles)
```

Comment:
Thanks for your pull request.  Since the issue states that this route should be unrestricted the only change needed would be to remove this line.  However I think it would be good to at least restrict this route to logged on users.  If a user is logged on that means that `req.user` will be set.  So instead of calling `authorizeByRole` I would just check if `req.user` is set, and if not throw the unauthorized error.

Reply:
@kenjiO  yes, it sounds better

Reply:
It has to be unrestricted because it's used to show the food preferences selector on the customer signup form

Reply:
@kenjiO I noticed that the route is already covered by requiresLogin?
https://github.com/freeCodeCamp/pantry-for-good/blob/5afd834dc2d4d2a1ac0f8ce10d5b6b727f60776d/server/routes/food.js#L23

Reply:
Yeah, you should have to add the GET /foods route before that line and remove `authorizeByRole` in the controller list function for it to work.